problems/arrayMethods.js

```diff
@@ -156,7 +243,14 @@ const arePositiveEvenAndNonZeroEnding = () => {};
  * @returns {string[]} All strings uppercase.
  */
 
-const stringsAndCaps = () => {};
+const stringsAndCaps = (items) => {
+    return items.map((el) => {
+      if (typeof el === "string") {
+      return el.toUpperCase();
+      }
+    })
+    .filter((el) => el ) 
```

Comment:
Personally I would filter first and then map. I think it makes understanding the code a bit more readable, though this will work great :-)